assignment-tool/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -11690,9 +11688,9 @@ private void initializeAssignment(SessionState state)
 			try {
 				st = SiteService.getSite((String) state.getAttribute(STATE_CONTEXT_STRING));
 				if (contentreviewSiteMin > 0 && st.getTitle().length() < contentreviewSiteMin){
-					addAlert(state, rb.getFormattedMessage("review.sitechars", new Object[]{contentreviewSiteMin, reviewServiceName}));
+					addAlert(state, rb.getFormattedMessage("review.sitechars", new Object[]{reviewServiceName, contentreviewSiteMin}));
```

Comment:
Think it was also because the site title length is known at that point so it makes sense to tell the user then, as it's just a warning it has no impact if they finally don't mark the CR option. It makes sense to me, it won't happen often anyway.